### PR TITLE
fix(site/pages): Fixed missing links for site pages

### PR DESCRIPTION
**Pull request by dmiller9911**
Fixed links in top right for guidelines, getting-started, and react-reference.

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -13,41 +13,48 @@ const plugins = [
...
     resolve: 'gatsby-source-filesystem',
     options: {
       path: `${PROJECT_ROOT}/src/patternfly`,
-      name: 'patternfly',
-    },
+      name: 'patternfly'
+    }
+  },
+  {
+    resolve: 'gatsby-source-filesystem',
```
Comment (dmiller9911) [inline gatsby-config.js]: Actual Change here.

File: gatsby-config.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
-const path = require('path')
+const path = require('path');
```
Comment (dgutride) [inline gatsby-config.js]: semicolons :)

File: gatsby-config.js [hunk 3]
```diff
@@ -13,41 +13,48 @@ const plugins = [
...
       precision: 5
-    },
+    }
   },
   {
     resolve: `gatsby-plugin-page-creator`,
     options: {
-      path: `${PROJECT_ROOT}/src/patternfly`,
+      path: `${PROJECT_ROOT}/src/patternfly`
     }
```
Comment (dgutride) [inline gatsby-config.js]: Can we address these linting issues with a better linter so your reviews aren't always fixing things?

Comment (dmiller9911) [reply gatsby-config.js]: Yeah.  That might be a good idea.  Are you opposed to using eslint with the same config that patternfly-react uses?

**Pull request merged**